### ISSUE TITLE
CC and BIE for new release does not show up properly.

### ISSUE THREAD

**Issue by kbserm** [OAGi/Score#930]
Currently in test.oagiscore.net, Line Base ACC has Last Modification Date Time added in 10.7. This is supposed to be inherited to Catalog Line. 

1. After published 10.7 release, it seems that Catalog ACC does not show up properly. Info around Line Base screwed up. See below.

![image](https://user-images.githubusercontent.com/7863166/95029973-19967680-067a-11eb-9898-b0fd0ba6a4f2.png)

2. Catalog BIE created based on 10.7 also screwed up.

Catalog BIE created for 10.6 looks fine.

**Comment by kbserm** [comment]
Tried on another one also suffered the similar issue. In this case, Sample Measure Range ASCCP has it ACC revised from Sample Measure Range Base to Sample Measure Range ACC. The ASCCP and corresponding BIE do not show up properly.

![image](https://user-images.githubusercontent.com/7863166/95032009-d7276680-0686-11eb-9d36-4b6a31b59395.png)

**Comment by rekuku** [comment]
Fixed. Make new release Draft/Publish and test please.

**Comment by kbserm** [comment]
@rekuku The problem is fixed for the catalog line case. 

Sample Measure Range it is okay in the CC but in the BIE it got worse, everything disappeared.

![image](https://user-images.githubusercontent.com/7863166/95262392-0a3e3700-07fa-11eb-8953-215f72abf2d4.png)

**Comment by rekuku** [comment]
10.7 Ascc/bcc has data relation problem that linked with working ASCCP/BCCP.
It happened during create release Draft.
Now it fixed and test server data also migrated.

**Comment by kbserm** [comment]
Looks fine now. Leave it opened for @asdimitriadis to create a test script that cover this. There are a few cases to cover.
1. Changing based type.
2. Changing BDT.
3. Adding ASCC.
4. Add BCC.
5. Moving ASCC.
6. Moving BCC.
7. #1 - #6 occurred in a based type.
8. #2 - #6 occurred in a group.

Basically, verified that BIE created based on old and new release does not and does have the expected content.

**Comment by asdimitriadis** [comment]
The above scenarios are tested. The only point that I have is the following:

I have revised the "Get BOM" and among other changes, I appended the ASCCP "Operation Group. Operation Group" (see first screenshot).
Then I created a new release.
After that I created a BIE using this revised "Get BOM". The resulting BIE is displayed in the second screenshot. The nodes of the Operation Group are attached to the root BIE node.

![Get Bom ASCCP with Operation Group appended (1)](https://user-images.githubusercontent.com/37205288/98982048-c4594a80-24ec-11eb-9159-b88ad2db1258.png)


![bie from the Get BOM (2)](https://user-images.githubusercontent.com/37205288/98982062-c7543b00-24ec-11eb-9387-269ff19840e7.png)
